src/mobile/styles/components/Orders.pcss

```diff
@@ -56,6 +56,70 @@
       }
     }
 
+    &__dropdown {
```

Comment:
oh...
Remove duplicating styles. Just change ```Dropdown.pcss``` for mobile, with additional class name if it's needed, but don't duplicate such things.